google/cloud/pubsub/internal/tracing_helpers.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUB_INTERNAL_TRACING_HELPERS_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUB_INTERNAL_TRACING_HELPERS_H
+
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+
+#include "google/cloud/pubsub/pull_ack_handler.h"
+#include "google/cloud/pubsub/version.h"
+#include "opentelemetry/trace/span_startoptions.h"
+
+namespace google {
+namespace cloud {
+namespace pubsub_internal {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+// Create start span options for a root span.
+opentelemetry::trace::StartSpanOptions RootStartSpanOptions();
```

Comment:
optional: Consider something like the following, which may be more composable:

```cc
opentelemetry::trace::StartSpanOptions MakeRootSpanOptions(opentelemetry::trace::StartSpanOptions o = {});
```

Reply:
I thought about that too. I don't think it's needed though. We can refactor later if there's a use case for it. 

Gonna add tests. Was hoping no one would notice...